Create de_DE.lang

Galaxy2Alex opened the pull request:
Translation is fully done and ready to be used. It's 90% playtested

Galaxy2Alex committed:
Create de_DE.lang

Galaxy2Alex committed:
Update de_DE.lang

(Just posting the raw file to prevent other people from starting a translation for nothing)

Galaxy2Alex committed:
Update de_DE.lang

Blocks

Galaxy2Alex committed:
Update de_DE.lang

Component Types

Galaxy2Alex committed:
Update de_DE.lang

Connection options/sets

Galaxy2Alex committed:
Update de_DE.lang

First part of Component Menus

Galaxy2Alex committed:
Update de_DE.lang

Done COMPONENT MENUS

Galaxy2Alex committed:
Update de_DE.lang

Completed w/o playtesting

Galaxy2Alex committed:
Update de_DE.lang

Galaxy2Alex committed:
Update de_DE.lang

Playtesting update

Galaxy2Alex committed:
Update de_DE.lang

Freaking Relay message...

Vexatos commented on src/assets/stevesfactory/lang/de_DE.lang:
```diff
@@ -0,0 +1,303 @@
+#				BLOCKS
+# ======================================
+
+tile.sfm.BlockMachineManager.name=Machineninventarmanager
```
"Maschineninventarmanager"; you forgot the "s" in "Maschine"

Vexatos commented on src/assets/stevesfactory/lang/de_DE.lang:
```diff
@@ -0,0 +1,303 @@
+# ======================================
+
+tile.sfm.BlockMachineManager.name=Machineninventarmanager
+tile.sfm.BlockCable.name=Invaentarkabel
```
"Inventarkabel"

Vexatos commented on src/assets/stevesfactory/lang/de_DE.lang:
```diff
@@ -0,0 +1,303 @@
+tile.sfm.BlockCableCreative.name=Kreativversorger
+tile.sfm.BlockCableIntake.name=Itemventil
+tile.sfm.BlockCableBUD.name=Blockdetector
+tile.sfm.BlockCableBreaker.name=Blockschranke
```
Wouldn't be "Blockgatter" much better?

Vexatos commented:
Actually, just let this being pulled. I will then make a typo fix Pull Request.
Mostly, @Galaxy2Alex, you did a pretty good job! Well done!

Galaxy2Alex committed:
Update de_DE.lang

Thanks @Vexatos for finding the errors/suggestions (yeah, "Gatter" is probably better, allthough we're not using it that much "down here" (Austria :P)

Galaxy2Alex commented:
Changes still apply to the pull request even if you make them after the request itself. Thanks :)

Vexatos commented:
I know it does, just did not want to look through the whole thing before I actually fix stuff, but, this way it works as well...
